pkg/install/install.go

```diff
@@ -383,41 +407,28 @@ func (i *Installer) initializeKubernetesClients(ctx context.Context) error {
 	return err
 }
 
-func (i *Installer) deployARMTemplate(ctx context.Context, rg string, tName string, t *arm.Template, params map[string]interface{}) error {
+func (i *Installer) deployARMTemplate(ctx context.Context, rg string, tName string, t *arm.Template, params map[string]interface{}) (err error) {
 	timeoutCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
 	defer cancel()
 
-	err := wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
```

Comment:
This is to avoid returning the timeout error, as we've starting doing elsewhere because it masks the preceding error